server-utils/src/lib.rs

```diff
@@ -15,5 +17,12 @@ pub mod hosts;
 pub mod session;
 pub mod reactor;
 mod matcher;
+mod stream_codec;
 
 pub use matcher::Pattern;
+
+/// Codecs utilities
+pub mod codecs {
```

Comment:
Just wanted all codecs (currently only one) it accessible via short namespace `codecs`